spec/factories/produce_item.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require 'faker'
+
+FactoryBot.define do
+  factory :produce_item do
+    sequence(:name) { Faker::Food.fruits }
```

Comment:
I feel like this is still susceptible for collisions. You probably should do something like:
```rb
sequence(:name) { |n| "#{Faker::Food.fruits}-#{n}" }
```

Reply:
edited as you suggested, but I'm going to read a bit on sequences in FactoryBot :]

Reply:
so sequence adds n(number) to our fruit name, that way each is unique right? Somehow I misunderstood it a bit at first.